ieeeuoft/newhacks-site-2022
# NH-30: add filters to orders api

## Overview

- Resolves #NH-30
- added filters that work and actually update the orders that are displayed


## Unit Tests Created

- none, i commented out a few to get it to work


## Steps to QA

- go to the orders page and make sure the filters and sorting work as they should
